node_modules/nodeos-barebones/node_modules/nodeos-cross-toolchain/scripts/build

```diff
@@ -112,6 +112,8 @@ if [[ ! -d $OBJ_DIR ]]; then
         --with-sysroot=$TOOLS/$TARGET         \
         --disable-nls                         \
         --disable-shared                      \
+        --disable-target-libsanitizer         \
+        --disable-libsanitizer                \
```

Comment:
Maybe is http://lists.busybox.net/pipermail/buildroot/2014-May/095861.html related? We need to investigate the purposse of libsanitizier and to document if it's important or not.